### ISSUE TITLE
Translate UI text for new strings

### ISSUE THREAD

**Issue by mmistakes** [mmistakes/minimal-mistakes#449]
As part of #424 several new text strings need translating in [`/_data/ui-text.yml`](https://github.com/mmistakes/minimal-mistakes/blob/master/_data/ui-text.yml). Pull requests welcome :smile:
- [x] Spanish
- [x] Brazilian Portuguese
- [x] Turkish
- [x] Italian
- [x] French

``` yaml
comments_title             : "Comments"
comment_form_info          : "Your email address will not be published. Required fields are marked"
comment_form_comment_label : "Comment"
comment_form_md_info       : "Markdown is supported."
comment_form_name_label    : "Name"
comment_form_email_label   : "Email address"
comment_form_website_label : "Website (optional)"
comment_btn_submit         : "Submit Comment"
comment_btn_submitted      : "Submitted"
comment_success_msg        : "Thanks for your comment! It will show on the site once it has been approved."
comment_error_msg          : "Sorry, there was an error with your submission. Please make sure all required fields have been completed and try again."
loading_label              : "Loading..."
```

**Comment by rschaerer** [comment]
In my opinion you would have a more relaxed :relaxed: time concerning the [_"by no means a full-on i18n solution"_](https://mmistakes.github.io/minimal-mistakes/docs/ui-text/) feature, if you would **add all the missing strings in English to the several different translations and append a comment like `# translation missing` to the corresponding lines** like so:

    # Turkish
    # -----------------
    tr: &DEFAULT_TR
      page                       : "Sayfa"
      [...]
      undefined_wpm              : "_config.yml dosyasında tanımlanmamış words_per_minute parametresi"
      comment_form_info          : "Your email address will not be published. Required fields are marked" # translation missing
      [...]

Otherwise the empty keys in the translations **which lead to omitting them from view might confuse users** which switch the language and didn't read the [docs](https://mmistakes.github.io/minimal-mistakes/docs/ui-text/) carefully.

How do you think about this? If you agree, I could do the work and create a PR for this.

**Comment by mmistakes** [comment]
My thought was empty keys fallback to English (what the theme currently does), which should hopefully be a clue when switching languages that they're missing. 

Example: `{{ site.data.ui-text[site.locale].comments_label | default: "Comments" }}`

Previously they were just blank and I noticed quite a few forks where half of the UI text was missing. So I added the English `default` values as a nudge to consult the docs for those who actually care about such things.

Open to suggestions.
